### PR TITLE
Stripe elements

### PR DESCRIPTION
**Pull request by jonahbron**
Starting to integrate Stripe elements

### CODE REVIEW

**Comment by jonahbron** [comment]
This PR is ready for review.  Would love a second pair of eyes on a bit of testing/looking at the code.

File: join-form-js/src/signup.ts [hunk 1]
```diff
@@ -238,36 +189,33 @@ export class Signup extends LitElement {
 
   @state()
   private availableRegions = allCountries.find(
-    (countryData) => countryData[1] === 'US'
+    (countryData) => countryData[1] === 'US',
```
Comment (stephenmcmurtry) [inline join-form-js/src/signup.ts:192]: Can we get/Do we need Canada support here?

**Review by stephenmcmurtry — COMMENTED** [review]
Looks generally good. How can I test the join form and worker together? Would be useful to add to the README

**Comment by jonahbron** [comment]
To run locally, open two terminals and run `npm start` in both the `join-form-js/` and `signup-worker/` directories

File: join-form-js/src/signup.ts (hunk 1 above)
Comment (jonahbron) [reply join-form-js/src/signup.ts:192]: Looking at the code, that's just the default.  mailingCountryChangeHandler updates this list.

**Comment by jonahbron** [comment]
@stephenmcmurtry I'd like to get this into QA so we can have some real users run through and turn up any blocking issues.  The  README says to talk to you about that :-)

**Comment by shiibainu** [comment]
@jonahbron @stephenmcmurtry ping for status

**Comment by jonahbron** [comment]
## Rollout plan

- [ ] Capture rollback point
  - [ ] Capture current signup worker version: [current](https://dash.cloudflare.com/394d48aba686b2466fe12e540c5d9466/workers/services/view/signup-worker/production/deployments)
  - [ ] Capture current join form: [current](https://dash.cloudflare.com/394d48aba686b2466fe12e540c5d9466/pages/view/join-form/077e15ee-a1e3-45e5-8110-b74172dee548)
- [ ] Deploy latest version
  - [ ] Deploy latest signup worker with `wrangler deploy`
  - [ ] Deploy latest join form with `wrangler deploy`
- [ ] Verify update
  - [ ] Run through signup in production
- [ ] Rollback or merge
  - If a blocking issue is encountered, switch to old version captured above and rollback
  - If no blocking issues are found, merge this PR
  - File issues in Github for any non-blocking issues

**Comment by jonahbron** [comment]
Testing observations:

- Server key needs to have `rak_setup_intent_read` permission
- User is told everything about their bank information goes to stripe; might be good to tell the user we don't receive or use any of that info, only use it to charge them
- Using incorrect credit card information allowed the flow to progress but with no payment method attached.

**Comment by shiibainu** [comment]
Couple of additional notes:
- One hiccup was needing to update the Webflow site to add the stripe payment container div
- ACHv2 means that if you go through the bank account flow, you also receive an email address with confirmation of addition (so the mandate, I guess?). Very fancy! Also confirmed that with ACHv2 configured, the billing portal does indeed allow self-update of payment method. Yay!
- The $5 init fee is now rolled into the first monthly dues charge, Matt is going to separately check whether the onboarding python script needs any updates to support this (maybe taking out one of the stripe checks)

Thanks so much Jonah!

**Comment by mattrubin** [comment]
Additional observations:
- To get the new form code working on the live AWU site, we needed to add an additional `<div slot="stripe-payment-container"></div>` inside of the `<awu-signup>` element.
- The Plaid API key and plaid-token-worker are no longer needed, and can be removed.
- We should verify that the code to automatically enable the dues subscription during onboarding works with the new subscription setup.
- We still need a way for members on ACHv1 to delete their old payment methods before adding a new one via the Stripe customer portal.

**Pull request merged**